browser-test/src/support/admin_programs.ts

```diff
@@ -58,6 +58,16 @@ export interface BlockSpec {
   questions: QuestionSpec[]
 }
 
+export interface ProgramSelectorSpec {
```

Comment:
```suggestion
export interface ProgramDefinitionSpec {
```
(blocking) Selector makes me think this is going to be related to a css selector or a playwright locator. Maybe `ProgramDefinitionSpec`? 

Eventually I want to get to having a full model that matches the backend datamodel, but that is a much much larger refactor.

Reply:
Thanks for the feedback! The issue is that it is a selector, not a definition... addProgramBlockUsingSpec is finding (selecting) a program and then adding a block to it, not creating (defining) the program. So for example if programs had a category, and you had to know the category to navigate to the program, we would add a category field to the selector so the code could find the program. We wouldn't want the selector to include fields like externalLink or confirmationMessage since you don't find a program based on those fields.

I'm fine to inline the interface too - see the comment below about that.